<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CQDlgVertexEdit</class>
 <widget class="QDialog" name="CQDlgVertexEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>230</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>361</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Vertex mode properties</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QCheckBox" name="qqShowHiddenVertices">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>161</width>
      <height>17</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Show hidden vertices</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>361</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>Operations on selected vertices</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QToolButton" name="qqInsertTriangles">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>161</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Insert triangles</string>
    </property>
   </widget>
   <widget class="QToolButton" name="qqInsertFan">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>161</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Insert triangle fan</string>
    </property>
   </widget>
   <widget class="QToolButton" name="qqMakeDummies">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>20</y>
      <width>161</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Make dummies</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>361</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>Vertex selection</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QToolButton" name="qqClearSelection">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>161</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Clear selection</string>
    </property>
   </widget>
   <widget class="QToolButton" name="qqInvertSelection">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>20</y>
      <width>161</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Invert selection</string>
    </property>
   </widget>
   <widget class="QLabel" name="qqInfo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>341</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>qqShowHiddenVertices</tabstop>
  <tabstop>qqClearSelection</tabstop>
  <tabstop>qqInvertSelection</tabstop>
  <tabstop>qqInsertTriangles</tabstop>
  <tabstop>qqInsertFan</tabstop>
  <tabstop>qqMakeDummies</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
